accumulo-store/src/main/java/gaffer/accumulostore/inputformat/ElementInputFormat.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2016 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gaffer.accumulostore.inputformat;
+
+import gaffer.accumulostore.key.AccumuloElementConverter;
+import gaffer.accumulostore.key.AccumuloKeyPackage;
+import gaffer.accumulostore.key.exception.AccumuloElementConversionException;
+import gaffer.commonutil.CommonConstants;
+import gaffer.data.element.Element;
+import gaffer.data.element.function.ElementTransformer;
+import gaffer.data.elementdefinition.exception.SchemaException;
+import gaffer.data.elementdefinition.view.View;
+import gaffer.data.elementdefinition.view.ViewElementDefinition;
+import gaffer.exception.SerialisationException;
+import gaffer.store.StoreException;
+import gaffer.store.schema.Schema;
+import org.apache.accumulo.core.client.mapreduce.InputFormatBase;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.util.format.DefaultFormatter;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.mapreduce.InputSplit;
+import org.apache.hadoop.mapreduce.RecordReader;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.Map;
+
+/**
+ *
+ */
+public class ElementInputFormat extends InputFormatBase<Element, NullWritable> {
+
+    public static final String KEY_PACKAGE = "KEY_PACKAGE";
+    public static final String SCHEMA = "SCHEMA";
+    public static final String VIEW = "VIEW";
```

Comment:
We have a VIEW and a SCHEMA key in AccumuloStoreConstants - perhaps we could reuse these?